Makefile

```diff
@@ -18,6 +18,8 @@ GATEKEEPER_VERSION = v3.10.0
 GATEKEEPER_IMAGE ?= ${RP_IMAGE_ACR}.azurecr.io/gatekeeper:$(GATEKEEPER_VERSION)
 GOTESTSUM = gotest.tools/gotestsum@v1.11.0
 
+OCP_RELEASE_BRANCH ?= 4.10
```

Comment:
>Do we need to inherit this value from the environment instead of pinning it to a specific version?

Using automation (running it from Makefile with a bash script) would be easier than manually updating the release branch variable in the bash script and then running it. Modifying it in the Makefile helps it be kept widely (and more apparent IMO) as it resides in the parent directory where other tools are used.

>For reproducibility reasons, shouldn't this be treated as any other dependency that we pin to specific values and bump up regularly for everyone 

It could but pinning could be messy, where using the OCP version seems easier to sustain and understand. Using makefile for this purpose could ease the process for any developer.